content/en-us/glossary/vocabulary.md

```diff
@@ -40,6 +43,12 @@ Words and definitions used across the IoT Atlas.
 ### Production
 > effort to mass produce devices and roll-out the IoT product to the market.
 
+### Programmable Logic Controller <a name="plc"></a>
+> ruggedized computing device or digital computer for industrial applications to monitor and control production/manufacturing processes ([Wikipedia]({{< ref "https://en.wikipedia.org/wiki/Programmable_logic_controller" >}})).
```

Comment:
Causing render error due to use of `ref`.

Wikipedia link should be standard markdown like this: `([Wikipedia](https://en.wikipedia.org/wiki/Programmable_logic_controller))`